src/Components/Namespace.ts

```diff
@@ -23,8 +23,10 @@ export class Namespace implements IComponentComposite {
         }
         result.push(declaration.join(''));
         this.parts.forEach((part: IComponentComposite): void => {
-            result.push(part.toPUML()
-            .replace(/^/gm, '    '));
+            result.push(
+                part.toPUML()
+                    .replace(/^(?!\s*$)/gm, '    ')
```

Comment:
What is this regular expression attempting to replace?

Reply:
Starting of line except empty lines. To add indentation.